### PR TITLE
Achievents APi

### PR DESCRIPTION
**Pull request by Surya1231**
Implemented and tested achievements API which fetches achievements for the currently logged student.

### CODE REVIEW

File: app/routes/user.router.js [hunk 1]
```diff
@@ -19,5 +19,6 @@ router.post('/updateProfile', jwtMiddleware.verify, authMiddleware.ensureLoggedI
 router.post('/changePassword', jwtMiddleware.verify, authMiddleware.ensureLoggedIn, userController.changePassword)
 router.get('/contributions', jwtMiddleware.verify, authMiddleware.ensureLoggedIn, userController.contributions)
 router.post('/updateBatch', jwtMiddleware.verify, authMiddleware.ensureOfficialPlacementTeam, userController.updateBatch)
+router.get('/achievements', jwtMiddleware.verify, authMiddleware.ensureLoggedIn, userController.achievements)
```
Comment (Pankajtanwarbanna) [inline app/routes/user.router.js:22]: We need to ensure that only student can access the achievements. It's any way there.

Comment (Surya1231) [reply app/routes/user.router.js:22]: ensureLoggedIn will enough for that I think

Comment (Pankajtanwarbanna) [reply app/routes/user.router.js:22]: Yeah, It will work but only students has achievements so **ensureStudent** would be more specific.

Comment (Surya1231) [reply app/routes/user.router.js:22]: But **spc** also have achievements and **ensureStudent** will not give permission to view achievements to SPCs.

Comment (Pankajtanwarbanna) [reply app/routes/user.router.js:22]: SPC will never have achievements. SPCs has two separate accounts one as normal student and another as SPC.

**Pull request merged**